content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -29,3 +29,27 @@ cluster must satisfy the following requirements:
   `app` label is used to add contextual information in distributed
   tracing. The `app` and `version` labels are also used to add contextual information
   in the metric telemetry collected by Istio.
+
+* _**`NET_ADMIN` capability**:_ The pods must have the `NET_ADMIN` capability allowed. The initialization containers of
+  the Envoy proxies require this capability. To check which capabilities are allowed for your pods, check if their
+  [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) can use a
+  [pod security policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) that allows the `NET_ADMIN`
+  capability.
```

Comment:
Sorry for the late comment.  You might want to add that the PodSecurityPolicy admission controller is not enabled in the kube-apiserver by default and this section only applies if it is enabled.  The commands in this section won't work unless the PodSecurityPolicy admission controller is setup (which is not done on basic kubeadm setups).

Reply:
Thanks @tiswanso . I have addressed it in https://github.com/istio/istio.io/pull/2891/commits/45d3402ae3021f8ef1d6cc672a29583c1cc99ad8